py/tests/unit/with_runtime_sparkling/test_conversions.py

```diff
@@ -78,15 +79,16 @@ def testFloatRDDToH2OFrame(spark, hc):
     rdd = spark.sparkContext.parallelize([0.5, 1.3333333333, 178])
     h2o_frame = hc.asH2OFrame(rdd)
     assert h2o_frame[0, 0] == 0.5
-    assert h2o_frame[1, 0] == 1.3333333333
+    assert pytest.approx(h2o_frame[1, 0]) == 1.3333333333
+    assert h2o_frame[2, 0] == 178
     unit_test_utils.asert_h2o_frame(h2o_frame, rdd)
 
 
 def testDoubleRDDToH2OFrame(spark, hc):
     rdd = spark.sparkContext.parallelize([0.5, 1.3333333333, 178])
     h2o_frame = hc.asH2OFrame(rdd)
     assert h2o_frame[0, 0] == 0.5
-    assert h2o_frame[1, 0] == 1.3333333333
+    assert pytest.approx(h2o_frame[1, 0]) == 1.3333333333
```

Comment:
What is the difference caused by this change?

Reply:
Haven't look into that, just fixed the test. I would say it's because we convert Spark RDD to DF on Python side and there might be some type conversion underneath.

The value is really same, but because it's double we can't compare it anymore exactly

Reply:
ok, thanks!